src/pages/provider/activityOverview.tsx

```diff
@@ -0,0 +1,178 @@
+import * as React from 'react';
+import { styled } from 'goober';
+import { useHashLocation } from '../../components/hash-router';
+import { Calendar, Edit3, Grid, MapPin, Plus, Trash2 } from 'react-feather';
+import {
+  Main,
+  Title,
+  ActivityCard,
+  CardWrapper,
+  ActivityImage,
+  CardTextContainer,
+  CardTitle,
+  CardIconContainer,
+  NewActivityCardTextContainer,
+  NewActivityTitle,
+  Header,
+  AddActivityButton,
+  HeaderNavigation,
+  NavItem,
+  Border,
+} from '../../components/forms/input';
+
+const styles = {
+  Header: styled(Header)`
+    display: flex;
+    position: relative;
+  `,
+  SubHeader: styled(Header)`
+    @media (max-width: 1023px) {
+      display: block;
+      position: relative;
+      margin-top: 48px;
+    }
+    @media (min-width: 1024px) {
+      display: flex;
+      justify-content: space-between;
+      position: relative;
+      margin-top: 48px;
+    }
+  `,
+  RightNavItem: styled(NavItem)`
+    padding: 12px 18px;
+  `,
+};
+
+/**
+ * Provider contact details form
+ * @returns
+ */
+export function ProviderActivityOverviewPage(): JSX.Element {
+  const [, navigate] = useHashLocation();
+
+  return (
+    <Main>
```

Comment:
Since this component is embedded inside of an Openstad site this Main component creates some strange looking layout:
<img width="1709" alt="Screenshot 2021-06-07 at 13 18 35" src="https://user-images.githubusercontent.com/1849831/121008091-e0774e80-c792-11eb-965a-e0744bc2f3f7.png">